docs/content-crag/guides/dagster/graph_job_op.mdx

```diff
@@ -710,7 +710,7 @@ def do_something_else(arg1):
     return arg1
 
 
-@graph(ins={"arg1": GraphIn(int)}, out=GraphOut(int))
+@graph(ins={"arg1": GraphIn()}, out=GraphOut())
```

Comment:
Is there any reason to provide a `GraphIn` and `GraphOut` here? They could just be omitted, right?

Reply:
Right. updated the GraphIn part.

But we can't omit GraphOut bc we can't easily tell if that's a multi out case at run time. According to the docstring, this is the similar case in `@op` where we also need to do `@op(out={'word': Out(), 'num': Out()})` - not sure if we plan to omit that on op too or have already done so. cc @dpeng817